File: 1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array.java
class Solution {
    public int minNumberOperations(int[] target) {
        int n = target.length;
        int count = target[0];
        for(int i = 1; i < n; ++i) count += Math.max(0, target[i] - target[i - 1]);
        return count;
    }
}

File: 1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array.cpp
class Solution {
public:
    int minNumberOperations(vector<int>& target) {
        int n = target.size();
        int count = target[n - 1];
        for(int i = n - 2; i >= 0; --i)
            count += max(0, target[i] - target[i + 1]);
        return count;
    }
};